apiserver/cmd/main.go

```diff
@@ -30,11 +31,20 @@ var (
 	rpcPortFlag        = flag.String("rpcPortFlag", ":8887", "RPC Port")
 	httpPortFlag       = flag.String("httpPortFlag", ":8888", "Http Proxy Port")
 	collectMetricsFlag = flag.Bool("collectMetricsFlag", true, "Whether to collect Prometheus metrics in API server.")
+	logFile            = flag.String("logFilePath", "", "sync log to local file")
 )
 
 func main() {
 	flag.Parse()
 
+	if *logFile != "" {
+		flagSet := flag.NewFlagSet(os.Args[0], flag.ExitOnError)
+		klog.InitFlags(flagSet)
+		_ = flagSet.Set("alsologtostderr", "true")
```

Comment:
Is `flagSet.Set("alsologtostderr", "true")` equivalent to `_ = flagSet.Set("alsologtostderr", "true")`? If so, maybe we can update it to the former one.

Reply:
flagSet.Set("alsologtostderr", "true") return error, but we assume these would not happen.

If i use `flagSet.Set("alsologtostderr", "true")` would lead to go format error in UT. = =

Reply:
Got it. Thank you for the explanation!